textacy/export.py

```diff
@@ -61,14 +65,35 @@ def docs_to_gensim(docs, lemmatize=True,
     """
     gdict = Dictionary()
     gcorpus = []
-    for doc in docs:
-        words = extract.words(doc,
-                              filter_stops=filter_stops,
-                              filter_punct=filter_punct,
-                              filter_nums=filter_nums)
+    stringstore = StringStore()
```

Comment:
StringStore is a spaCy class that very efficiently maps strings to integer IDs. When a new string is encountered, it automatically assigns it to the next available integer. It's used here because we want to re-map strings' spaCy vocab IDs, which were generated from a different corpus and usually result in large gaps between consecutive strings IDs, such that the strings in our corpus are "densely" represented. Does that make sense...?